src/frontend/src/tests/btc/components/fee/UtxosFeeContext.spec.ts

```diff
@@ -0,0 +1,150 @@
+import UtxosFeeContext from '$btc/components/fee/UtxosFeeContext.svelte';
+import * as btcSendApi from '$btc/services/btc-send.services';
+import {
+	initUtxosFeeStore,
+	UTXOS_FEE_CONTEXT_KEY,
+	type UtxosFeeStore
+} from '$btc/stores/utxos-fee.store';
+import { BTC_MAINNET_NETWORK_ID, ICP_NETWORK_ID } from '$env/networks.env';
+import * as authStore from '$lib/derived/auth.derived';
+import * as authServices from '$lib/services/auth.services';
+import { mockUtxosFee } from '$tests/mocks/btc.mock';
+import { mockIdentity } from '$tests/mocks/identity.mock';
+import { mockPage } from '$tests/mocks/page.store.mock';
+import type { Identity } from '@dfinity/agent';
+import { render, waitFor } from '@testing-library/svelte';
+import { readable } from 'svelte/store';
+import { expect } from 'vitest';
+
+describe('UtxosFeeContext', () => {
+	const amount = 10;
+	const networkId = BTC_MAINNET_NETWORK_ID;
+	const mockContext = (store: UtxosFeeStore) => new Map([[UTXOS_FEE_CONTEXT_KEY, { store }]]);
+	const mockBtcSendApi = () =>
+		vi.spyOn(btcSendApi, 'selectUtxosFee').mockResolvedValue(mockUtxosFee);
+	const mockAuthStore = (value: Identity | null) =>
+		vi.spyOn(authStore, 'authIdentity', 'get').mockImplementation(() => readable(value));
+
+	const props = {
+		amount,
+		networkId
+	};
+
+	beforeEach(() => {
+		mockPage.reset();
```

Comment:
reset the utxos store as well on before?

Reply:
yep, done